app/services/search_indexer.rb

```diff
@@ -31,7 +31,7 @@ def self.refresh_filterable_properties(client_ids = nil, limit = 1000)
           Client.where(id: client_ids)
         end
 
-      attributes = Client.where(id: client_ids).includes(:tax_returns, :documents, intake: :dependents).map do |client|
+      attributes = Client.includes(:tax_returns).where(id: client_ids).map do |client|
```

Comment:
dont we look up documents & intake for each client, which is why we `include(:tax_returns, :documents, :intake)`, not sure about the dependents piece?